src/graphql/queries/metadata.graphql

```diff
@@ -93,3 +93,25 @@ query getRewardsDataForWallet($address: String!) {
     }
   }
 }
+
+query reverseResolveENSProfile(
```

Comment:
not using it just yet, but well, we have more fucntionality from backend, so I figured out I will add it since it was somewhat related. We might switch to that for most reverse record fetching soon